### PR TITLE
py - add warning when saving incorrectly named screenshot

### PR DESCRIPTION
**Pull request by cgoldberg**
this addresses #4108 for Python bindings.

- [X ] By placing an `X` in the preceding checkbox, I verify that I have signed the [Contributor License Agreement](https://github.com/SeleniumHQ/selenium/blob/master/CONTRIBUTING.md#step-6-sign-the-cla)

### CODE REVIEW

File: py/selenium/webdriver/remote/webelement.py [hunk 1]
```diff
@@ -450,6 +452,9 @@ def screenshot(self, filename):
             return False
         finally:
             del png
+        if not filename.lower().endswith('.png'):
```
Comment (AutomatedTester) [inline py/selenium/webdriver/remote/webelement.py]: Can you move this warning to be the first thing that happens in the method so people can see the warning _and_ get any exceptions that might happen.

File: py/selenium/webdriver/remote/webdriver.py [hunk 2]
```diff
@@ -842,15 +844,20 @@ def get_screenshot_as_file(self, filename):
             return False
         finally:
             del png
+        if not filename.lower().endswith('.png'):
```
Comment (AutomatedTester) [inline py/selenium/webdriver/remote/webdriver.py]: Can you move this warning to be the first thing that happens in the method so people can see the warning _and_ get any exceptions that might happen.

File: py/selenium/webdriver/remote/webelement.py (hunk 1 above)
Comment (cgoldberg) [inline py/selenium/webdriver/remote/webelement.py]: done

File: py/selenium/webdriver/remote/webdriver.py (hunk 2 above)
Comment (cgoldberg) [inline py/selenium/webdriver/remote/webdriver.py]: done

**Comment by abotalov** [comment]
Do you think saving an image to a path without extension should be fine as well?

**Comment by AutomatedTester** [comment]
> Do you think saving an image to a path without extension should be fine as well?

I think this is fine. If people do silly things like that then thats their own fault.

**Pull request merged**